voice/src/commonMain/kotlin/handlers/UdpLifeCycleHandler.kt

```diff
@@ -22,18 +23,18 @@ internal class UdpLifeCycleHandler(
     flow: Flow<VoiceEvent>,
     private val connection: VoiceConnection
 ) : ConnectionEventHandler<VoiceEvent>(flow, "UdpInterceptor") {
-    private var ssrc: UInt? by atomic(null)
-    private var server: InetSocketAddress? by atomic(null)
+    private val ssrc = atomic<UInt?>(null)
+    private val server = atomic<SocketAddress?>(null)
 
     private var audioSenderJob: Job? by atomic(null)
 
     @OptIn(ExperimentalUnsignedTypes::class)
     override suspend fun start() = coroutineScope {
         on<Ready> {
-            ssrc = it.ssrc
-            server = InetSocketAddress(it.ip, it.port)
+            ssrc.value = it.ssrc
+            server.value = SocketAddress(it.ip, it.port)
 
-            val ip: InetSocketAddress = connection.socket.discoverIp(server!!, ssrc!!.toInt())
+            val ip: SocketAddress = connection.socket.discoverIP(server.value!!, ssrc.value!!.toInt())
```

Comment:
While testing, `ssrc` would be `null` here, this fixes it 🤷🏼

Reply:
the atomic delegate is broken?